src/txn/txn_rollback_to_stable.c

```diff
@@ -1173,6 +1284,13 @@ __rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
     WT_ASSERT(session, F_ISSET(session, WT_SESSION_LOCKED_SCHEMA));
     WT_RET(__wt_metadata_cursor(session, &cursor));
 
+    if (F_ISSET(S2C(session), WT_CONN_RECOVERING))
+        __wt_verbose(session, WT_VERB_RECOVERY_RTS(session),
+          "Recovered checkpoint snapshot min  %" PRIu64 ", snapshot max : %" PRIu64
```

Comment:
Don't give space for the name and `:` also `:` is missed for `snapshot min`.

Reply:
Fixed